src/main/scala/fpinscala/answers/iomonad/DerivingIO.scala

```diff
@@ -527,15 +523,13 @@ object IO3:
   // A specialized state monad
   case class ConsoleState[A](run: Buffers => (A, Buffers)):
     def map[B](f: A => B): ConsoleState[B] =
-      ConsoleState { s =>
+      ConsoleState: s =>
```

Comment:
Perhaps this is discussed elsewhere: if `ConsoleState` was a `newtype` wrapper, rather than a `case class`, would his call to the constructor be needed?